parquet-column/src/test/java/org/apache/parquet/schema/TestTypeBuilders.java

```diff
@@ -155,39 +155,18 @@ public void testFixedTypeConstruction() {
 
   @Test
   public void testEmptyGroup() {
-    assertThrows("Should complain that required group is empty",
-        IllegalStateException.class, new Callable<Type>() {
-          @Override
-          public Type call() throws Exception {
-            return Types.requiredGroup().named("g");
-          }
-        });
-    assertThrows("Should complain that optional group is empty",
-        IllegalStateException.class, new Callable<Type>() {
-          @Override
-          public Type call() throws Exception {
-            return Types.optionalGroup().named("g");
-          }
-        });
-    assertThrows("Should complain that repeated group is empty",
-        IllegalStateException.class, new Callable<Type>() {
-          @Override
-          public Type call() throws Exception {
-            return Types.repeatedGroup().named("g");
-          }
-        });
+    Assert.assertEquals("Should not complain about an empty required group",
```

Comment:
Can we add comment on why this should be supported so next time it will not be accidentally removed